lib/holidays.rb

```diff
@@ -161,6 +161,22 @@ def to_weekday_if_boxing_weekend(date)
       DateCalculatorFactory.weekend_modifier.to_weekday_if_boxing_weekend(date)
     end
 
+    def to_tuesday_if_sunday_or_monday_if_saturday(year)
```

Comment:
For future reference, I am considering whether these methods need to actually be 'public'. I think with my large refactor of 'custom methods' we can reference the factory directly in the YAML files. The only reason these are publicly available is because of how custom methods used to work.

I definitely don't want you to change this because this is how it works today, just wanted you to be aware that I might be changing this in the near future. End result will be the same. 😄